### PR TITLE
Updates some npm packages to fix linting and vulnerabilities

### PR DESCRIPTION
**Pull request by tinkertrain**
This PR updates:

- Typescript ESlint plugin: I noticed that linting was not working after some internal dependency got updated, they fixed the issue and I updated the package.
- Brings all storybook dependencies to latest
- Updates nyc from an ancient version riddled with lots of vulnerabilities to latest (coverage maintained at 100%)
- Updates babel svg plugin to latest, had to adjust the babel configuration a bit

In a nutshell, goes from:

`44 vulnerabilities (4 low, 3 moderate, 37 high)`

to

`6 vulnerabilities (2 low, 2 moderate, 2 high)`

The remaining issues are from dependencies that we already have at latest.

Tests and coverage all pass, storybook runs smoothly

### CODE REVIEW

**Comment by tjbo** [comment]
Awesome work, looks good to me! 🚀
